stories/topics/proc-aws-marketplace.adoc

```diff
@@ -9,39 +9,22 @@ Follow these steps to purchase and deploy the public offer.
 .Procedure
 
 . In the AWS Console, navigate to menu:AWS Marketplace Subscriptions[].
-. Click btn:[Discover Products].
+. In the lefthand navigation bar, click btn:[Discover Products].
```

Comment:
We should avoid using phrases like 'lefthand' side. They're frowned upon by accessibility standards.